packages/sheets/src/services/sheet-interceptor/__tests__/sheet-interceptor.service.spec.ts

```diff
@@ -89,6 +90,37 @@ describe('Test SheetInterceptorService', () => {
             expect(getCell(0, 0)).toEqual({ v: 'intercepted' });
             expect(getCell(0, 1)).toEqual({ v: 'A2' });
         });
+
+        it('should skip some interceptors by filtering out the effect', () => {
+            const interceptorService = get(SheetInterceptorService);
+            interceptorService.intercept(INTERCEPTOR_POINT.CELL_CONTENT, {
+                id: 'should-skip',
+                priority: 100,
+                handler(_cell, location: ISheetLocation, next: (v: Nullable<ICellData>) => Nullable<ICellData>) {
+                    if (location.row === 0 && location.col === 0) {
+                        return { v: 'intercepted A' };
+                    }
+
+                    return next();
+                },
+            });
+
+            interceptorService.intercept(INTERCEPTOR_POINT.CELL_CONTENT, {
+                id: 'should-not-skip',
+                priority: 0,
+                handler(_cell, location: ISheetLocation, next: (v: Nullable<ICellData>) => Nullable<ICellData>) {
+                    if (location.row === 0 && location.col === 0) {
+                        return { v: 'intercepted B' };
+                    }
+
+                    return next();
+                },
+            });
+
+            expect(getCell(0, 0)).toEqual({ v: 'intercepted A' });
+            expect(getCell(0, 1)).toEqual({ v: 'A2' });
+            expect(getCell(0, 0, 'should-skip', (interceptor) => interceptor.id !== 'should-skip')).toEqual({ v: 'intercepted B' });
```

Comment:
the key is use for cache the intercepter gourp,  the real effect code is filter call back , so please do not use key as Id, it may make other mistake